app/controllers/surveys_controller.rb

```diff
@@ -1,4 +1,35 @@
 class SurveysController < ApplicationController
+
   def send_survey
```

Comment:
True, forgot about that 👍 Would be cool if we could alias so the url is `surveys/:id/send` but it calls `send_survey`. I believe this would be through specifying a `path` options on the routes file but not 100% sure. Up to you if you want to dig around the docs!